### PR TITLE
Lint vc generator

### PR DESCRIPTION
**Pull request by aljones15**

### CODE REVIEW

File: tests/vc-generator/TestLinkedDataSignature.js [hunk 1]
```diff
@@ -92,15 +88,12 @@ export class LinkedDataSignature extends LinkedDataProof {
...
 
   /**
-   * @param document.document
-   * @param {object} document - Document to be signed.
-   * @param purpose - {ProofPurpose}.
-   * @param documentLoader - {function}.
-   * @param expansionMap - {function}.
+   * @param {object} options - Options to use.
+   * @param {object} options.document - Document to be signed.
+   * @param {object} options.purpose -  A JSON-LD purpose of the proof.
```
Comment (dlongley) [inline tests/vc-generator/TestLinkedDataSignature.js:93]:
```suggestion
   * @param {object} options.purpose -  The purpose of the proof.
```

File: tests/vc-generator/TestLinkedDataSignature.js [hunk 2]
```diff
@@ -92,15 +88,12 @@ export class LinkedDataSignature extends LinkedDataProof {
...
+   * @param {Function} options.documentLoader - A JSON-LD document loader.
+   * @param {Function} options.expansionMap - A JSON-LD Expansion map.
    *
-   * @param document.purpose
-   * @param document.documentLoader
-   * @param document.expansionMap
    * @returns {Promise<object>} Resolves with the created proof object.
    */
   async createProof({document, purpose, documentLoader, expansionMap}) {
```
Comment (dlongley) [inline tests/vc-generator/TestLinkedDataSignature.js:99]: `expansionMap` is deprecated and should probably just be removed here.

File: tests/vc-generator/TestLinkedDataSignature.js [hunk 3]
```diff
@@ -158,30 +151,18 @@ export class LinkedDataSignature extends LinkedDataProof {
...
+   * @param {object} options.document - The document the proof applies to.
+   * @param {Function} options.documentLoader - A JSON-LD document loader.
+   * @param {Function} options.expansionMap - A JSON-LD expansion map.
    *
-   * @param proof.document
-   * @param proof.documentLoader
-   * @param proof.expansionMap
    * @returns {Promise<{object}>} Resolves with the verification result.
    */
   async verifyProof({proof, document, documentLoader, expansionMap}) {
```
Comment (dlongley) [inline tests/vc-generator/TestLinkedDataSignature.js:168]: `expansionMap` could be removed here and anywhere else.

**Review by dlongley — APPROVED** [review]
See comments. Thanks!

**Review by JSAssassin — APPROVED** [review]

File: tests/vc-generator/TestLinkedDataSignature.js (hunk 2 above)
Comment (aljones15) [reply tests/vc-generator/TestLinkedDataSignature.js]: thanks for the note updated `@db/vc`, `jsonld`, and `jsigs` and then removed the expansionMap. https://github.com/w3c-ccg/di-ed25519-test-suite/pull/32/commits/202312f0bd378638bc38281b278fbbceb5d74d7a

File: tests/vc-generator/TestLinkedDataSignature.js (hunk 3 above)
Comment (aljones15) [reply tests/vc-generator/TestLinkedDataSignature.js]: expansionMap removed here: https://github.com/w3c-ccg/di-ed25519-test-suite/pull/32/commits/202312f0bd378638bc38281b278fbbceb5d74d7a

**Review by dlongley — APPROVED** [review]

**Pull request merged**